patches/chromium/gl_add_support_for_eglgetplatformdisplayext_extension.patch

```diff
@@ -0,0 +1,365 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Caner Altinbasak <caltinbasak@brightsign.biz>
+Date: Mon, 21 Oct 2024 14:57:01 +0100
+Subject: gl: Add support for eglGetPlatformDisplayExt extension
+
+This change is needed for Thor platform which requires eglGetPlatformDisplayExt
+for wayland backend
+
+diff --git a/ui/gl/egl_bindings_autogen_mock.cc b/ui/gl/egl_bindings_autogen_mock.cc
+index 8c56ef8caf0c99c92168351f7c54ce5756470ba4..4917a5a9e5541849a9137749b834fd018f3350e0 100644
+--- a/ui/gl/egl_bindings_autogen_mock.cc
++++ b/ui/gl/egl_bindings_autogen_mock.cc
+@@ -416,6 +416,15 @@ MockEGLInterface::Mock_eglGetPlatformDisplay(EGLenum platform,
+   return interface_->GetPlatformDisplay(platform, native_display, attrib_list);
+ }
+ 
++EGLDisplay GL_BINDING_CALL
++MockEGLInterface::Mock_eglGetPlatformDisplayExt(EGLenum platform,
++                                                void* native_display,
++                                                const EGLAttrib* attrib_list) {
++  MakeEglMockFunctionUnique("eglGetPlatformDisplayExt");
++  return interface_->GetPlatformDisplayExt(platform, native_display,
++                                           attrib_list);
++}
++
+ __eglMustCastToProperFunctionPointerType GL_BINDING_CALL
+ MockEGLInterface::Mock_eglGetProcAddress(const char* procname) {
+   MakeEglMockFunctionUnique("eglGetProcAddress");
+@@ -799,10 +808,9 @@ static void MockEglInvalidFunction() {
+ 
+ GLFunctionPointerType GL_BINDING_CALL
+ MockEGLInterface::GetGLProcAddress(const char* name) {
+-  if (strcmp(name, "eglAcquireExternalContextANGLE") == 0) {
```

Comment:
Was this change put in on purpose?

Reply:
I've used the code autogenerator bundled in chromium itself:
ui/gl/generate_bindings.py

I don't know quite why there are changes in the generated code.